pkgs/development/python-modules/pyfronius/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pyfronius";
+  version = "0.5.2";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    pname = "PyFronius";
+    inherit version;
+    sha256 = "077h6qh3c3dp6wdd64137m1lwp8v74k1qqi6l5cc0wgrsq8b0m1x";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+  ];
+
+  # Project has not published tests yet
```

Comment:
https://github.com/nielstron/pyfronius/tree/master/pyfronius/tests

Reply:
Yes, there are tests which are not part of the source published on PyPI.

https://github.com/nielstron/pyfronius/issues/26

Reply:
Why not fetchFromGitHub? Also does ha has tests we can enable?

Reply:
> Why not fetchFromGitHub?

no tags

> Also does ha has tests we can enable?

no: https://github.com/home-assistant/core/tree/2021.6.6/tests/components

Reply:
Maybe the comment could state that the tests are not published on PyPI and GitHub has no tags?

Reply:
Upstream has added a tag yesterday. Thus, now GitHub is used and the tests are running.